src/js/button.js

```diff
@@ -38,24 +38,27 @@ class Button extends Component {
 
     let el = super.createEl(type, props);
 
-    // if innerHTML hasn't been overridden (bigPlayButton), add content elements
-    if (!props.innerHTML) {
-      this.contentEl_ = Dom.createEl('div', {
-        className: 'vjs-control-content'
-      });
+    this.controlTextEl_ = Dom.createEl('span', {
+      className: 'vjs-control-text'
+    });
 
-      this.controlText_ = Dom.createEl('span', {
-        className: 'vjs-control-text',
-        innerHTML: this.localize(this.buttonText) || 'Need Text'
-      });
+    el.appendChild(this.controlTextEl_);
 
-      this.contentEl_.appendChild(this.controlText_);
-      el.appendChild(this.contentEl_);
-    }
+    this.controlText();
 
     return el;
   }
 
+  controlText(text) {
+    if (!text && (this.controlTextEl_ && this.controlTextEl_.innerHTML)) return this.controlText_ || 'Need Text';
```

Comment:
Note: Change this to a simple setter/getter and pass the text into the controlText() call above.